app/validators/blocks/list_collector_validator.py

```diff
@@ -6,23 +6,17 @@
 
 class ListCollectorValidator(BlockValidator, ValidateListCollectorQuestionsMixin):
     LIST_COLLECTOR_KEY_MISSING = "Missing key in ListCollector"
-    ADD_ANSWER_REFERENCE_NOT_IN_MAIN_BLOCK = (
-        "add_answer reference uses id not found in main block question"
-    )
+    REDIRECT_TO_LIST_ADD_BLOCK = "RedirectToListAddBlock"
+    REMOVE_LIST_ITEM_AND_ANSWERS = "RemoveListItemAndAnswers"
```

Comment:
We should add a validation to ensure only `PrimaryPersonListCollector` can use `RemoveListItemAndAnswers` as we only support it in that.